### ISSUE TITLE
Weird pixel particle-looking things in Blue Sea Dragon

### ISSUE THREAD

**Issue by theflyingzamboni** [Legend-of-Dragoon-Modding/Severed-Chains#1810]
These appear to exist in retail, but blend different so that there aren't dark spots on the ice. This may be related to #1308.

![image](https://github.com/user-attachments/assets/c7f4d21b-d955-42f5-a83d-65b15ff20549)

**Comment by dragoonsouls** [comment]
SC is capable of Particle Pixel Rendering with the new Rendering?... i was checking the code and the Pixel Particle and seems to be not touch for a long time...

**Comment by dragoonsouls** [comment]
RENDERER.queueOrthoModel(RENDERER.opaqueQuad, this.mv, QueuedModelStandard.class) is queueing an Ortho model...

**Comment by LordMonoxide** [comment]
Yes it supports pixel particles, but it appears that it's supposed to have B+F translucency and this was lost at some point before 2.1

3dfca266253613981573c26332827e124028e6a3
